compiler/one-cmds/utils.py

```diff
@@ -138,6 +138,9 @@ def _parse_cfg(args, driver_name):
 def _make_tf2tfliteV2_cmd(args, driver_path, input_path, output_path):
     """make a command for running tf2tfliteV2.py"""
     cmd = [sys.executable, os.path.expanduser(driver_path)]
+    # verbose option
+    if args.verbose:
```

Comment:
How about use this function for consistency?
```suggestion
    if _is_valid_attr(args, 'verbose'):
```

Reply:
Sure.